deviceModeInit.js

```diff
@@ -85,15 +85,26 @@ var rudderTracking = (function () {
     });
 
     switch (name) {
+      /**
+       * We are actually not tracking collections but rather products
+       * /collections/{collectionName}/products --> This is the actual product list 
+       * that we intend to track
+       */
+      case "/products":
+      case "/collections/":
+        // TODO: check if something is there after collections/ in the URL
+        // otherwise it a collections page and we dont track that
+        if (isProductListPage()) {
+          productListPage(val);
+        }
+
       case "/products/":
-        console.log("/products/");
+        // TODO: check if something is there after products/ in the URL
```

Comment:
What are we doing to address the TODO item we should consider that

Reply:
Yes have updated it